portal/src/main/webapp/js/src/patient-view/util/ClinicalAttributesUtil.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+/**
+ * Functions for dealing with clinical attributes.
+ */
+window.ClinicalAttributesUtil = (function() {
+  /**
+   * Clean clinical attributes. Useful for rounding numbers, or other types of
+   * data cleanup steps. Probably differs per institution.
+   * @param {object} clinicalData - key/value pairs of clinical data
+   */
+  function clean(clinicalData) {
+    // Shallow Copy clinicalData
+    var cleanClinicalData = $.extend({}, clinicalData);
+    var NULL_VALUES = [
+      'not applicable',
+      'not available',
+      'pending',
+      'discrepancy',
+      'completed',
+      '',
+      'null',
+      'unknown',
+      'na',
+      'n/a'
+    ];
```

Comment:
also include:
- [unknown]
- [not submitted]
- [not evaluated]

These might also be helpful to include although @zheins has been doing some post-processing with tcga data (where most of these attr values appear):
- [not applicable]
- [not available]
- [undefined]